### PR TITLE
Added sample notebooks, badges to docker README and minor changes to helper.

### PR DESCRIPTION
**Pull request by kiddinn**
This PR adds the following things:

1. Sample notebooks
2. Badges to docker README to explain how to setup a docker container
3. Ability to select sections in the TOML file for access configuration
4. New options for the data frame flattening

### CODE REVIEW

File: jupyter/laceworkjupyter/__init__.py [hunk 1]
```diff
@@ -33,12 +33,14 @@ class LaceworkJupyterHelper:
 
     def __init__(
             self, api_key=None, api_secret=None, account=None,
-            subaccount=None, instance=None, base_domain=None):
+            subaccount=None, instance=None, base_domain=None,
+            config_section=None):
```
Comment (alannix-lw) [inline jupyter/laceworkjupyter/__init__.py:37]: nit. Can we change `config_section` to `profile` to match the terminology from the Lacework CLI? https://github.com/lacework/go-sdk/tree/main/cli#multiple-profiles
```suggestion
            profile=None):
```

File: laceworksdk/api/__init__.py [hunk 2]
```diff
@@ -60,7 +60,8 @@ def __init__(self,
                  api_key=None,
                  api_secret=None,
                  instance=None,
-                 base_domain=None):
+                 base_domain=None,
+                 config_section=None):
```
Comment (alannix-lw) [inline laceworksdk/api/__init__.py:64]: Same as above.
```suggestion
                 profile=None):
```

File: laceworksdk/api/__init__.py [hunk 3]
```diff
@@ -88,7 +89,7 @@ def __init__(self,
             os.path.expanduser('~'), LACEWORK_CLI_CONFIG_RELATIVE_PATH)
 
         if os.path.isfile(config_file_path):
-            config_section = os.getenv(
+            config_section = config_section or os.getenv(
```
Comment (alannix-lw) [inline laceworksdk/api/__init__.py:92]: Same as above.
```suggestion
            config_section = profile or os.getenv(
```

**Review by alannix-lw — CHANGES_REQUESTED** [review]

File: jupyter/laceworkjupyter/__init__.py (hunk 1 above)
Comment (kiddinn) [reply jupyter/laceworkjupyter/__init__.py]: done

File: laceworksdk/api/__init__.py (hunk 2 above)
Comment (kiddinn) [reply laceworksdk/api/__init__.py]: done

File: laceworksdk/api/__init__.py (hunk 3 above)
Comment (kiddinn) [reply laceworksdk/api/__init__.py]: done

**Review by kiddinn — COMMENTED** [review]
done

**Review by alannix-lw — APPROVED** [review]

**Pull request merged**